apps/deploy-web/src/services/auth/auth/auth.service.ts

```diff
@@ -0,0 +1,63 @@
+import type { HttpClient } from "@akashnetwork/http-sdk";
+import type { InternalAxiosRequestConfig } from "axios";
+
+import { ANONYMOUS_USER_TOKEN_KEY } from "@src/config/auth.config";
+import { ONBOARDING_STEP_KEY } from "../../storage/keys";
+
+export class AuthService {
+  constructor(
+    private readonly urlService: AuthUrlService,
+    private readonly internalApiHttpClient: HttpClient,
+    private readonly location = window.location,
+    private readonly localStorage = window.localStorage
+  ) {}
+
+  async signup(options?: { returnTo?: string }): Promise<void> {
+    const params = new URLSearchParams();
+    if (options?.returnTo) params.append("returnTo", options.returnTo);
+
+    // send http request to store anonymous user token in a cookie
+    // this is is needed only when anonymous free trial is enabled
```

Comment:
```suggestion
    // this is only needed when anonymous free trial is enabled
```

Reply:
It requires reconfiguration of unleash on frontend and I just don’t want to do it in this PR. It seems like too much for such a simple thing

Reply:
Ok I see makes sense